cypress/e2e/04-conduct_test.cy.js

```diff
@@ -76,6 +76,9 @@ describe("Conducting a COVID test", () => {
     // then it won't trigger a network call
     cy.wait("@GetFacilityQueue", {timeout: 20000});
 
+    // Wait for the FacilityQueue results to populate
```

Comment:
could we add a link to Zedd's comment in your other PR so we can remember to update this once Mike's work gets in?